src/jdk.jshell/share/classes/jdk/internal/jshell/tool/resources/l10n_de.properties

```diff
@@ -167,7 +167,7 @@ jshell.console.completion.all.completions.number = <Erneut Tabulatortaste dr\u00
 jshell.console.completion.all.completions = <Erneut Tabulatortaste dr\u00FCcken, um alle m\u00F6glichen Abschl\u00FCsse anzuzeigen>
 jshell.console.no.javadoc = <Keine Dokumentation gefunden>
 jshell.console.do.nothing = Nichts machen
-jshell.console.choice = Auswahl: 
+jshell.console.choice = Auswahl:\u0020
```

Comment:
This is correct.  However, the original l10n.properties did not get the change as intended.

l10n.properties had it implemented as 

`jshell.console.choice = Choice: \`

The trailing white space was enclosed with a trailing `\` (probably to signify the white space is intentional) and thus hidden from the script.

It should be changed to `jshell.console.choice = Choice:\u0020` for consistency